src/test/unit/gov/nist/javax/sip/parser/ims/PAssertedIdentityParserTest.java

```diff
@@ -41,5 +42,29 @@ public void testParser() {
 
         super.testParser(PAssertedIdentityParser.class,preferredID);
     }
+    
+    public void testInvalidHeaderParam() {
+    	
+        String[] preferredIDs =  {
```

Comment:
for this particular error cases, i would prefer to have different tests. One like "testInvalidHeaderParam",  other like "testInvalidHdrParamOnMultiValue".... 
Take into account the positive cases are using the array/loop approach,because there is a framework for that. So either create a framework for invalid cases, or just use plain/single test cases,ok?